### PR TITLE
LPS-75462 adding option 'none' to auto complete

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by linolaoj** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-text/src/main/java/com/liferay/dynamic/data/mapping/type/text/internal/TextDDMFormFieldTypeSettings.java [hunk 1]
```diff
@@ -83,8 +83,8 @@
 
 	@DDMFormField(
 		label = "%create-list",
-		optionLabels = {"%manually", "%from-data-provider"},
-		optionValues = {"manual", "data-provider"},
+		optionLabels = {"%manually", "%from-data-provider", "%none"},
+		optionValues = {"manual", "data-provider", "none"},
```
Comment (rafaprax) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-text/src/main/java/com/liferay/dynamic/data/mapping/type/text/internal/TextDDMFormFieldTypeSettings.java]: Hey @linolaoj What do you think if the value for the `none` option is an empty String `""` ?

Comment (linolaoj) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-text/src/main/java/com/liferay/dynamic/data/mapping/type/text/internal/TextDDMFormFieldTypeSettings.java]: @rafaprax , I think empty string is good. I'm going to push this change.

Comment (linolaoj) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-text/src/main/java/com/liferay/dynamic/data/mapping/type/text/internal/TextDDMFormFieldTypeSettings.java]: done :+1:

**Comment by rafaprax** [comment]
LGTM

**Comment by pedroqueiroz94** [comment]
@linolaoj , I will close this until you send the version we talked about. Thanks a lot!

**Pull request closed without merging**